hack/generated-templates/updater-template.yaml

```diff
@@ -1,147 +1,71 @@
 apiVersion: v1
 kind: Template
 parameters:
-- name: ALLOWED_CIDR_BLOCKS
-  required: true
-- name: REGISTRY_IMG
-  required: true
-- name: IMAGE_TAG
-  required: true
-- name: REPO_NAME
-  value: private-cluster-rhapi-apischeme-updater
-  required: true
-metadata:
-  name: private-cluster-rhapi-apischeme-updater-template
+- {name: ALLOWED_CIDR_BLOCKS, required: true}
```

Comment:
Where did the curly braces come from this isn't valid yaml right?

Reply:
Yes, yaml allows you to represent non-nested dicts and lists inline for brevity. This big delta to the generated yaml is coming about with the change in versions (I assume it's due to the version of oyaml you get by default with py3, because we're not explicitly constraining the version of oyaml). If we wanted to keep this the way it was, we could add `default_flow_style=False` to our `yaml.dump` call.

Reply:
Thanks for the tip :) Fixed in https://github.com/openshift/private-cluster-rhapi-apischeme-updater/pull/3

Reply:
FWIW, I prefer the terser representation. But I also sympathize with minimizing deviation from other copies of this tooling. At least until my master plan comes to fruition, and we're maintaining generate_template.py in just one place...